<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PENG_NavigationWidget</class>
 <widget class="QWidget" name="PENG_NavigationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QLineEdit" name="leProject">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabPnid">
      <property name="toolTip">
       <string>P&amp;ID List</string>
      </property>
      <attribute name="icon">
       <iconset resource="peng.qrc">
        <normaloff>:/images/icons/pnid.png</normaloff>:/images/icons/pnid.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>P&amp;&amp;ID List</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabEquipment">
      <property name="toolTip">
       <string>Equipment List</string>
      </property>
      <attribute name="icon">
       <iconset resource="peng.qrc">
        <normaloff>:/images/icons/equipment.png</normaloff>:/images/icons/equipment.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>Equipment List</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QToolButton" name="pbAddEquipment">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Add equipment</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/images/icons/add.png</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pbDeleteEquipment">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Delete equipment</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../db/src/gui/db.qrc">
             <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pbSaveEquipment">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save to database</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../db/src/gui/db.qrc">
             <normaloff>:/images/icons/database_save.png</normaloff>:/images/icons/database_save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="cbEquipmentType">
           <property name="toolTip">
            <string>Equipment type</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tvEquipment">
         <property name="toolTip">
          <string>Equipment</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPipeLine">
      <property name="toolTip">
       <string>Line List</string>
      </property>
      <attribute name="icon">
       <iconset resource="peng.qrc">
        <normaloff>:/images/icons/pipeline.png</normaloff>:/images/icons/pipeline.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>Line List</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QToolButton" name="pbAddLine">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Add pipe line</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/images/icons/add.png</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pbDeleteLine">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Delete pipe line</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../db/src/gui/db.qrc">
             <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pbSaveLine">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save pipe line</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../db/src/gui/db.qrc">
             <normaloff>:/images/icons/database_save.png</normaloff>:/images/icons/database_save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="cbLineType">
           <property name="toolTip">
            <string>Pipe line type</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tvLine">
         <property name="toolTip">
          <string>Pipe line</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInstrument">
      <property name="toolTip">
       <string>Instrument List</string>
      </property>
      <attribute name="icon">
       <iconset resource="peng.qrc">
        <normaloff>:/images/icons/instrument.png</normaloff>:/images/icons/instrument.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>Instrument List</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabElectrical">
      <property name="toolTip">
       <string>Electrical Equipment List</string>
      </property>
      <attribute name="icon">
       <iconset resource="../../../db/src/gui/db.qrc">
        <normaloff>:/images/icons/lightning.png</normaloff>:/images/icons/lightning.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <attribute name="toolTip">
       <string>Electrical Equipment List</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../db/src/gui/db.qrc"/>
  <include location="peng.qrc"/>
 </resources>
 <connections/>
</ui>
